### PR TITLE
update deps

### PR DESCRIPTION
**Pull request by bwp91**
## :recycle: Current situation

*Describe the current situation. Explain current problems, if there are any. Be as descriptive as possible (e.g., including examples or code snippets).*

## :bulb: Proposed solution

*Describe the proposed solution and changes. How does it affect the project? How does it affect the internal structure (e.g., refactorings)?*

## :gear: Release Notes

*Provide a summary of the changes or features from a user's point of view. If there are breaking changes, provide migration guides using code examples of the affected features.*

## :heavy_plus_sign: Additional Information
*If applicable, provide additional context in this section.*

### Testing

*Which tests were added? Which existing tests were adapted/changed? Which situations are covered, and what edge cases are missing?*

### Reviewer Nudging

*Where should the reviewer start? what is a good entry point?*

### CODE REVIEW

File: src/server.ts [hunk 1]
```diff
@@ -38,8 +38,10 @@ export class HomebridgePluginUiServer {
     }
 
     process.addListener('message', (request) => {
```
Comment (dgreif) [inline src/server.ts]: It looks like the type for `request` switched from `any` to `unknown`, which makes it more restrictive on how you can use it without manually casting. If you want to simply get back to the old behavior, you can specify the type as `any` here. That should clear both of the ts errors in this function.

```suggestion
    process.addListener('message', (request: any) => {
```

File: src/server.ts [hunk 2]
```diff
@@ -85,6 +87,7 @@ export class HomebridgePluginUiServer {
           return this.sendResponse(request, { message: e.message, error: e.requestError }, false);
         } else {
           console.error(e);
+          // @ts-ignore
           return this.sendResponse(request, { message: e.message }, false);
```
Comment (dgreif) [inline src/server.ts]: This one is similar, but `e` used to be assumed to be an `Error` (I think) and is now `unknown`. You can do an inline cast here if you are ok assume it's an `Error`

```suggestion
          return this.sendResponse(request, { message: (e as Error).message }, false);
```

**Review by donavanbecker — APPROVED** [review]
